VideoLocker/src/main/java/org/edx/mobile/view/dialog/AlertDialogFragment.java

```diff
@@ -29,9 +32,18 @@ public static AlertDialogFragment newInstance(String message) {
     @NonNull
     @Override
     public Dialog onCreateDialog(Bundle savedInstanceState) {
-        return new AlertDialog.Builder(getContext())
-                .setMessage(getArguments().getString(ARG_MESSAGE))
+        String title = getArguments().getString(ARG_TITLE);
+        String message = getArguments().getString(ARG_MESSAGE);
+
+        AlertDialog ret = new AlertDialog.Builder(getContext())
```

Comment:
@mdinino Shorthand comes from this: https://robots.thoughtbot.com/sed-102-replace-in-place

:laughing: